src/GSF.CarbonIntensity/src/Handlers/ForecastHandler.cs

```diff
@@ -0,0 +1,59 @@
+using CarbonAware.Aggregators.CarbonAware;
+using CarbonAware.Exceptions;
+using CarbonAware.Tools.WattTimeClient;
+using CarbonAware.Tools.WattTimeClient.Configuration;
+using GSF.CarbonIntensity.Exceptions;
+using GSF.CarbonIntensity.Models;
+using Microsoft.Extensions.Logging;
+
+namespace GSF.CarbonIntensity.Handlers;
+
+internal sealed class ForecastHandler : IForecastHandler
+{
+    private readonly ICarbonAwareAggregator _aggregator;
+    private readonly ILogger<ForecastHandler> _logger;
+
+    public ForecastHandler(ILogger<ForecastHandler> logger, ICarbonAwareAggregator aggregator)
+    {
+        _logger = logger ?? throw new ArgumentNullException(nameof(logger));
+        _aggregator = aggregator ?? throw new ArgumentNullException(nameof(aggregator));
+    }
+
+    public async Task<EmissionsForecast?> GetCurrentAsync(string location, DateTimeOffset? start, DateTimeOffset? end, int? duration)
```

Comment:
Tested with JSON and WattTime data source.

The start, end and duration parameters should be optional. When I make the following call:

var result = await _handler.GetCurrentAsync(location);

I get the following error:

There is no argument given that corresponds to the required formal parameter 'start' of 'IForecastHandler.GetCurrentAsync(string, DateTimeOffset?, DateTimeOffset?, int?)'

Reply:
@esgraham, great, will fix it. Thanks for the earlier feedback.

Reply:
Bug fix pushed in